core/utils.lua

```diff
@@ -388,73 +388,93 @@ function serialize_string(s)
 	return string.format("%q", s)
 end
 
---#region palettes
-G.SETTINGS.selected_colours = G.SETTINGS.selected_colours or {}
+--#region alt textures
+G.SETTINGS.selected_texture = G.SETTINGS.selected_texture or {}
 G.PALETTE = {}
```

Comment:
is this needed?

Reply:
The saved settings don't exist without this and the game won't load, unsure if there's a cleaner way around this